mobile_verifier/src/owner_shares.rs

```diff
@@ -100,16 +190,17 @@ impl OwnerShares {
                     .into_iter()
                     .map(move |radio_share| proto::RadioRewardShare {
                         owner_key: owner_key.clone().into(),
-                        hotspot_key: radio_share.hotspot_key.into(),
                         cbsd_id: radio_share.cbsd_id,
                         amount: {
-                            let rewards = rewards_per_share * radio_share.amount;
+                            let rewards = rewards_per_share * radio_share.amount
+                                + transfer_rewards.reward(&radio_share.hotspot_key);
                             let rewards = (rewards * Decimal::from(MOBILE_SCALE))
```

Comment:
arent rewards already in bones and thus already at 10^6, i dont think MOBILE_SCALE is required here